### ISSUE TITLE
There have a BUG with StripesTransportPipe with AutoWorkbench!

### ISSUE THREAD

**Issue by Antecer** [BuildCraft/BuildCraft#2711]
1.Use StripesTransportPipe with Shears to shear sheep
2.Use AutoWorkbench to crafting any blocks.
3.when the AutoWorkbench craft a block it Throw a shears too!

this is my screenshots:
![bug](https://cloud.githubusercontent.com/assets/8684916/7610146/26bc9662-f9ad-11e4-8a7a-16c7eeaabd5c.png)
![qq 20150513195815](https://cloud.githubusercontent.com/assets/8684916/7610145/26bc64f8-f9ad-11e4-9039-8e7b25581cae.png)

**Comment by 3TUSK** [comment]
I confirmed that this dupe bug can be reproduced under Forge 10.13.2.1355, with only BC 6.4.15 installed:
![2015-05-13_17 51 55](https://cloud.githubusercontent.com/assets/9113033/7621953/caa8fd20-f998-11e4-9205-116bf78baed0.png)

Beside, this bug seems not work with bucket and other tools.
Furthermore, I set up two structure, and the screenshot is as shown:
![2015-05-13_17 58 02](https://cloud.githubusercontent.com/assets/9113033/7622061/c9f9a004-f999-11e4-915f-1c0f3d5eb957.png)
The thing is, auto-workbench on the left can only duplicate one shear, while the auto-workbench on the right can dupe shears constantly, even though it is "far" away from the stripe pipe which contains shear.
I cannot tell where has the bug exactly, but I believe it is more about auto-workbench.
